content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,186 @@
+the :guilabel:`Employee` and the :guilabel:`Work Entry Type` from the respective drop-down lists.
+
+.. image:: work_entries/create.png
+   :align: center
+   :alt: Create a new work entry by clicking Create and entering the information.
+
+Next, enter the date and time for the work entry in the :guilabel:`From` and :guilabel:`To`
+drop-downs. First, select the date by navigating to the correct month and year using the
+:guilabel:`arrow icons` icons, then click on the specific day.
+
+.. image:: work_entries/date.png
+   :align: center
+   :alt: Select the date from the calendar using the drop down arrow.
+
+Then, select the time by clicking on the :guilabel:`clock icon`, and using the :guilabel:`up and
+down arrows` for each section to enter the hour, minute, and second for the time period.
+
+.. image:: work_entries/time.png
+   :align: center
+   :alt: Enter the time in hours, minutes, and seconds, using the arrows.
+
+The :guilabel:`Period` will display the hours based on the :guilabel:`To` and :guilabel:`From`
+entries.
+
+.. tip::
+   Before clicking either :guilabel:`Save & Close` or :guilabel:`Save & New`, it is good to double
+   check the :guilabel:`Period` to ensure the time stated corresponds to the :guilabel:`To` and
+   :guilabel:`From` fields.
+
+   .. image:: work_entries/period.png
+      :align: center
+      :alt: Check the hours entered by looking at the total time in the Period section.
+
+Once the information is entered, click :guilabel:`Save & Close` to save the entry and close the
+pop-up, or :guilabel:`Save & New` to save the entry and create another :guilabel:`Work Entry Type`.
+
+Regenerate work entries
+=======================
+
+After a work entry has been either added or modified, the work entries need to be regenerated for
+the affected employee(s). Click on the :guilabel:`Regenerate Work Entries` button, and a pop-up will
+appear.
+
+   .. image:: work_entries/regenerate.png
+      :align: center
+      :alt: Regenrate work entires after modifying them.
+
+Select the :guilabel:`Employee` to regenerate work entries for from the :guilabel:`drop-down menu`,
+and adjust the :guilabel:`From` and :guilabel:`To` fields so the correct date range is displayed.
+Click the :guilabel:`Regenerate Work Entries` button and the work entries will be recreated. Once
+finished, the pop-up will close.
+
+   .. image:: work_entries/regenerate-details.png
+      :align: center
+      :alt: Regenerate a work entry for a particular employee.
+
+Conflicts
+=========
+
+A conflict appears for any request that has not been approved, such as sick time or vacation, or if
+there are any errors on the work entry, such as required fields being left blank. Conflicts are
+required to be resolved before payslips can be generated.
+
+Any work entry that has a conflict to be resolved is indicated on the main :guilabel:`Work Entry`
+overview dashboard, or by going to :menuselection:`Payroll --> Work Entries --> Conflicts`, where
```

Comment:
don't need the conjunction here since the navigational instruction is still part of that original clause
```suggestion
overview dashboard, located by going to :menuselection:`Payroll --> Work Entries --> Conflicts`, where
```